web/pgadmin/browser/server_groups/servers/databases/schemas/tables/static/js/table.ui.js

```diff
@@ -136,6 +136,20 @@ export class ConstraintsSchema extends BaseUISchema {
           }));
           return {columns: state.columns};
         }
+        /* If column or primary key is deleted */
+        if(actionObj.type === SCHEMA_STATE_ACTIONS.DELETE_ROW) {
+          let deletedColumn = _.differenceBy(actionObj.oldState.columns,state.columns,'name');
```

Comment:
A column can be renamed. By this logic, the renamed column will be considered as deleted.

Reply:
@adityatoshniwal  In the case of column rename the action is `set_value` so this condition will not get called. I have tested by renaming a column and this condition was skipped.

Reply:
we have a key called `cid` which is used internally to identify unique rows. please use `cid` instead of `name`